provisionassessmentimagesbucket_policy.tf

```diff
@@ -0,0 +1,44 @@
+# ------------------------------------------------------------------------------
+# Create the IAM policy that allows provisioning of the assessment images bucket
+# in the Images (Production) and Images (Staging) accounts.
+# ------------------------------------------------------------------------------
+
+data "aws_iam_policy_document" "provision_bucket_production" {
+  statement {
+    actions = [
+      "s3:*",
```

Comment:
I can adjust it, but I was following the way it was done in [cisagov/cool-dns-certboto](https://github.com/cisagov/cool-dns-certboto/blob/develop/provisioncertificatesbucket_policy.tf#L6-L18). Although it is a wildcard action, it is limited to a specific bucket per https://github.com/cisagov/cool-images-assessment-images/blob/ca57fd308985405439247e84f34d99ebbe42d487/provisionassessmentimagesbucket_policy.tf#L12-L14

Reply:
I think since he is creating the bucket and limiting the resource the `s3:*` is OK here.  This is how I have been doing it, so I'll need to make some corrections if we decide we don't like this.

Reply:
Even when it's limited to our a bucket created here, I still think there's benefit in specifying the permissions, rather than using a wildcard.  I don't think there are all that many S3 permissions needed here, are there?  I'm ok with leaving the wildcard if you guys are, but I'd prefer to see us working towards eliminating the `*`s where we can.

Reply:
I have opened https://github.com/cisagov/cool-images-assessment-images/issues/2 to narrow the permissions on this policy.